### PR TITLE
Adding "see also" with link to dotnet cli repo

### LINKED ISSUE #24105: This page should have a link to the GH repo where the CLI is at
which is https://github.com/dotnet/sdk.   

The product feedback link takes us to a dev comm space, which isnt at all useful for reporting defects, making PR's, etc

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b5b0701a-2043-4ac8-567a-03c1e35161b2
* Version Independent ID: 60d50636-8086-8d92-9ccc-60b7cecd714a
* Content: [dotnet sln command - .NET CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-sln)
* Content Source: [docs/core/tools/dotnet-sln.md](https://github.com/dotnet/docs/blob/main/docs/core/tools/dotnet-sln.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by StingyJack**
Fixes #24105

### CODE REVIEW

**Review by Youssef1313 — COMMENTED** [review]
Why this change would be relevant to `dotnet sln` only?

File: docs/core/tools/dotnet-sln.md [hunk 1]
```diff
@@ -204,3 +204,7 @@ dotnet sln [<SOLUTION_FILE>] remove [-h|--help]
   The following screenshot shows the result in Visual Studio 2019 **Solution Explorer**:
 
   :::image type="content" source="media/dotnet-sln/dotnet-sln-solution-folder.png" alt-text="Solution Explorer showing class library projects grouped into a solution folder.":::
+
+## See also
+
+- [dotnet/sdk GitHub repo](https://github.com/dotnet/sdk) (dotnet cli source)
```
Comment (tdykstra) [inline docs/core/tools/dotnet-sln.md:210]:
```suggestion
- [dotnet/sdk GitHub repo](https://github.com/dotnet/sdk) (.NET CLI source)
```

**Review by tdykstra — APPROVED** [review]
Thanks, looks good, one suggestion.

**Comment by adegeo** [comment]
Please don't merge this. @gewarren is using it to look at the "Check for build warnings"

**Comment by Youssef1313** [comment]
@adegeo @gewarren As a note here in case it's unnoticed. The OpenPublishing.Build says: `Validation status: passed`, but clicking on `details` shows a ton of warnings.

**Comment by gewarren** [comment]
The problem is that the old status isn't getting cleared out - presumably because there's no new commit. I may have to change the status checker to fail itself instead of inserting a new status. Or to always insert a new status - whether it's pass or fail.

File: docs/core/tools/dotnet-sln.md (hunk 1 above)
Comment (StingyJack) [reply docs/core/tools/dotnet-sln.md:210]: Is that the current way to refer to the dotnet.exe?

Comment (tdykstra) [reply docs/core/tools/dotnet-sln.md]: Depends on context and what specifically is referred to.  For the file name, *dotnet.exe*; as a tool set, `.NET CLI`

**Comment by StingyJack** [comment]
None of the warnings on that openpublishing check are for this file.

**Comment by adegeo** [comment]
Thanks @StingyJack

**Pull request merged**